src/pages/blog/[...page].astro

```diff
@@ -0,0 +1,99 @@
+---
```

Comment:
dynamic page template for when pagination kicks in (yes, this is a bit of future proofing but after 12 articles, i will add in the pagination component)